pyproject.toml

```diff
@@ -207,3 +217,52 @@ filterwarnings = [
     "ignore:unclosed file <_io.TextIOWrapper name='(/dev/null|nul)' mode='w':ResourceWarning", # OK
     "ignore:Deprecated in traitlets 4.1, use the instance .metadata dictionary directly", # OK (ipywidgets internal)
 ]
+
+[tool.mypy]
+python_version = "3.10"
```

Comment:
I think this is automatically inferred by `requires-python`

Reply:
I've removed the line.